libsql/src/replication/client.rs

```diff
@@ -58,7 +59,12 @@ impl Client {
             .try_into()
             .context("Invalid auth token must be ascii")?;
 
-        let ns = split_namespace(origin.host().unwrap()).unwrap_or_else(|_| "default".to_string());
+
+        let ns = maybe_namespace.unwrap_or_else(||
+            split_namespace(origin.host().unwrap())
+            .unwrap_or_else(|_| "default".to_string())
+        );
```

Comment:
this code is slightly hard to read, could we use `else {}` maybe?

Reply:
you're right, rewritten to more expressive form